templates/Elegant/index.tmpl

```diff
@@ -71,14 +71,6 @@
   [websvn-endtest]
     </table>
   [websvn-test:treeview]
-    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/group-collapse.js"></script>
-    [websvn-test:opentree]
```

Comment:
Same like before, you seemed to have removed some feature.

Reply:
i tried to move this ope-tree selection in JS but couldnt succeed. So I have reverted it back. This i will contribute or take care in separate issue. For now i have reverted it back with collapsible.js and added the call of jquery-min version 1.9.1 file on top of this line.